docs/release-notes/RELEASE-NOTES-2.1.0.md

```diff
@@ -7,6 +7,9 @@ It:
   * uses proper data directory.
 * Fixes `syncing` field in `/status` endpoint.
 * Adds `sync_progress` field to `/status` external endpoint (`sync_progress` metric is also reported every 30 seconds during sync process).
+* Exposes logged events (as the field `log`) in ContractCallObject in the HTTP API.
+* Improves stability of state channel's FSM
```

Comment:
I'd finish all points with the same punctuation mark. Most are finished with `.`, two are not (not worth job re-start this time though 🙂 ).